DataFormats/Detectors/TRD/include/DataFormatsTRD/DcsCcdbObjects.h

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file DcsCcdbObjects.h
+/// \brief Objects which are created from DCS DPs and stored in the CCDB
+/// \author Ole Schmidt, ole.schmidt@cern.ch
+
+#ifndef ALICEO2_DCSCCDBOBJECTSTRD_H
+#define ALICEO2_DCSCCDBOBJECTSTRD_H
+
+#include "DataFormatsTRD/Constants.h"
+#include "Rtypes.h"
+#include <array>
+
+namespace o2
+{
+namespace trd
+{
+
+struct TRDDCSMinMaxMeanInfo {
```

Comment:
I think this will be a read-only object and then it's a struct so there are no getters needed for the members. In case we would like to add getters later I believe that this does not change the ROOT streamer information and thus would still be possible. Only if we add/remove members then the streamer information would need to be updated